pkg/fission-cli/cmd/spec/apply.go

```diff
@@ -123,6 +124,8 @@ func (opts *ApplySubCommand) run(input cli.Input) error {
 			}
 		}
 
+		warnIfDirtyWorkTree(filepath.Clean(specDir + "/.."))
```

Comment:
If this function returns an error we should log it here, but not fail the entire program. This helps us in making sure this function is working properly from a future maintenance perspective.